tests/telemetry/test_writer.py

```diff
@@ -427,6 +434,7 @@ def test_app_client_configuration_changed_event(telemetry_writer, test_agent_ses
         ]
 
 
+@flaky(1735812000)
 def test_add_integration_disabled_writer(telemetry_writer, test_agent_session):
```

Comment:
ooof do we need to skip all these telemetry tests? Do we know when they started failing and with what error messages. If I have time, I would like to take a deeper look

Reply:
Maybe this can help you (dd-trace-py org)
https://app.datadoghq.com/ci/test-runs?query=test_level%3Atest%20%40test.status%3Afail%20%40test.suite%3Atest_writer.py&index=citest&start=1704657844200&end=1704744244200&paused=false

Reply:
Telemetry should be better after PR 8018 but it's still not perfect. I'm still working on it, maybe you can revert changes on that file for this PR ?